### ISSUE TITLE
[Adaptive Cards - Home Page] - At 400% reflow settings 320px viewport, auto playing video content is not playing and visible on the page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9316]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)

**Pre-requisites:**

1. Navigate using keyboard only.
2.  Go to system Settings-> System-> Display-> Scale & Layout-> Change the size of text, apps, and other items at 100%
3. Go to browser Settings-> Zoom- 200%
4. Change the view port of the page into 320px.

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to all the auto playing video content on the page using tab key.
3. Verify that, at 400% reflow settings 320px viewport, auto playing video content is playing and visible on the page or not.

**Actual Result:**
At 400% reflow settings 320px viewport, auto playing video content is not playing and visible on the page.

**Expected Result:**
At 400% reflow settings 320px viewport, auto playing video content should be properly playing and available visible on the page.

**User Experience:**
Low vision users depend on the reflow settings will be impacted if at 400% reflow settings 320px viewport, auto playing video content is not playing and visible on the page due to which users will not be able to know the information present in the video content and may be confused as some of the video content is not playing on the page. 

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/reflow.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/aa1fcc3b-484e-4594-b395-cbbfd8a64611

### Card JSON

```JSON
NA
```
